Testscripts/Linux/TVM-TestSecureBootCompatibility.sh

```diff
@@ -8,17 +8,49 @@
 }
 
 UtilsInit
-homeDir=$(dirname "$0")
 
-if [ ! -e "$homeDir"/sbinfo ]; then
-    LogErr "sbinfo tool is not on the system"
+GetDistro
+case $DISTRO in
+    redhat*|centos*|oracle*)
+        echo "[packages-microsoft-com-azurecore]" | tee -a /etc/yum.repos.d/azurecore.repo
+        echo "name=packages-microsoft-com-azurecore" | tee -a /etc/yum.repos.d/azurecore.repo
+        echo "baseurl=https://packages.microsoft.com/yumrepos/azurecore/" | tee -a /etc/yum.repos.d/azurecore.repo
+        echo "enabled=1" | tee -a /etc/yum.repos.d/azurecore.repo
+        echo "gpgcheck=0" | tee -a /etc/yum.repos.d/azurecore.repo
+    ;;
+    ubuntu*|debian*)
+        echo "deb [arch=amd64] http://packages.microsoft.com/repos/azurecore/ trusty main" | sudo tee -a /etc/apt/sources.list.d/azure.list
+        echo "deb [arch=amd64] http://packages.microsoft.com/repos/azurecore/ xenial main" | sudo tee -a /etc/apt/sources.list.d/azure.list
+        echo "deb [arch=amd64] http://packages.microsoft.com/repos/azurecore/ bionic main" | sudo tee -a /etc/apt/sources.list.d/azure.list
+
+        wget https://packages.microsoft.com/keys/microsoft.asc
+        wget https://packages.microsoft.com/keys/msopentech.asc
+
+        apt-key add microsoft.asc
+        apt-key add msopentech.asc
+    ;;
+    suse*|opensuse*|sles*|sle_hpc*)
+        zypper ar -t rpm-md -n "packages-microsoft-com-azurecore" --no-gpgcheck https://packages.microsoft.com/yumrepos/azurecore/ azurecore
+    ;;
+    *)
+        LogErr "Distro not supported. Aborting..."
+        UpdateSummary "Distro not supported. Aborting..."
+        SetTestStateAborted
+        exit 0
+    ;;
+esac
```

Comment:
can this be added to utils? can another tool vtpminfo  download/install like this 'azure-security'?